src/views/forms/NewMarketForm/v7/NewMarketPreviewStep.tsx

```diff
@@ -91,21 +120,35 @@ export const NewMarketPreviewStep = ({
     };
   }, [errorMessage, freeCollateral, stringGetter]);
 
-  const heading = shouldHideTitleAndDescription ? null : (
-    <>
-      <h2>{stringGetter({ key: STRING_KEYS.CONFIRM_LAUNCH_DETAILS })}</h2>
-      <span tw="text-color-text-0">
-        {stringGetter({
-          key: STRING_KEYS.DEPOSIT_LOCKUP_DESCRIPTION,
-          params: {
-            NUM_DAYS: <span tw="text-color-text-1">30</span>,
-            PAST_DAYS: 30,
-            APR_PERCENTAGE: <MegaVaultYieldOutput tw="inline-block" />,
-          },
-        })}
-      </span>
-    </>
-  );
+  const heading = useMemo(() => {
```

Comment:
Any particular reason we want to memo here? It's probably fine without, can just replace with a runFn() which I added for places where it's nice to have an inline function like this.